packages/flutter/test/material/app_test.dart

```diff
@@ -1602,6 +1602,20 @@ void main() {
     expect(find.byType(AnimatedTheme), findsNothing);
     expect(find.byType(Theme), findsOneWidget);
   });
+
+  testWidgets('MaterialApp works in an unconstrained environment', (WidgetTester tester) async {
+    // Regression test for https://github.com/flutter/flutter/issues/137875.
```

Comment:
nit: I'd move this above the `testWidgets` and reduce the number of empty lines in the body of the test.